superx/routes/signup.py

```diff
@@ -2,48 +2,42 @@
 from routes import home
 from flask_wtf import FlaskForm
 from wtforms import StringField, PasswordField, BooleanField
-from wtforms.validators import InputRequired,  Length
+from wtforms.validators import InputRequired, Length
 from flask import render_template, redirect, url_for, request
 from werkzeug.security import generate_password_hash, check_password_hash
 from models import *
 from flask_login import LoginManager, UserMixin, login_user, logout_user, current_user
 
-class LoginForm(FlaskForm):
-    email = StringField('email', validators=[InputRequired(), Length(min=3, max=50)])
-    password = PasswordField('password', validators=[InputRequired(), Length(min=6, max=80)])
 
-class RegisterForm(FlaskForm):
-    email = StringField('email', validators=[InputRequired(), Length(max=50)])
-    username = StringField('username', validators=[InputRequired(), Length(min=3, max=10)])
-    password = PasswordField('password', validators=[InputRequired(), Length(min=6, max=80)])
+class LoginForm(FlaskForm):
+    email = StringField('אימייל', validators=[InputRequired(), Length(min=3, max=50)])
+    password = PasswordField('סיסמה', validators=[InputRequired(), Length(min=6, max=80)])
 
 
+class RegisterForm(FlaskForm):
+    email = StringField('אימייל', validators=[InputRequired(), Length(max=50)], )
```

Comment:
You may want to consider using an email validator: https://wtforms.readthedocs.io/en/master/validators/